include/fem-dem/cfd_dem_coupling.h

```diff
@@ -152,7 +163,7 @@ class CFDDEMSolver : public GLSVANSSolver<dim>
 
 
   unsigned int                coupling_frequency;
-  bool                        contact_detection_step = false;
+  bool                        contact_detection_step = true;
```

Comment:
I don't understand why contact_detection_step should be initially set to true.

Reply:
Don't give default value in the .h. If you want to give it a value, give it in the constructor....